ValveResourceFormat/Resource/ResourceTypes/PostProcessing.cs

```diff
@@ -70,11 +70,11 @@ public byte[] GetRAWData()
             var j = 0;
             for (var i = 0; i < lut.Length; i++)
             {
-                // Skip each 4th byte
-                if (((i + 1) % 4) == 0)
-                {
-                    continue;
-                }
+                // Skip each 4th byte (this doesn't change anything)
```

Comment:
This makes the whole loop redundant. Does this still work correctly for the exporting?

Reply:
Fixed!